core/src/test/java/com/github/jsonldjava/specs/JsonLdApiTests.java

```diff
@@ -0,0 +1,66 @@
+package com.github.jsonldjava.specs;
```

Comment:
This clas is almost identical to the JsonLd1Tests. Can the code be reused ?

Reply:
I took a look at this, but I think avoiding the redundancy here would lead to too much complexity. First, the actual code is already extracted in static methods of SuiteUtils. The redundant part is only the JUnit `Parameters`/`BeforeClass`/`AfterClass` setup. These are static methods, so simply extracting them to a superclass won't work. An option would be a custom JUnit Rule (see https://www.baeldung.com/junit-4-rules#defining-a-custom-junit-rule), but I'm not sure if/how that would work with `Parameterized`, and I think it might actually make the code less readable.